wiki/Storyboard/Scripting/Audio/de.md

```diff
@@ -1,36 +1,36 @@
----
-outdated_translation: true
-outdated_since: 3edcc9733800c0e9221a2032eede25009608e79d
----
+# Storyboard-Audio
 
-# Storyboard Audio
-
-**Audio Dateien** (WAV, MP3, und OGG) können abgespielt werden. Es ist wie beim Deklarieren von Objekten, keine Kommandos, sie werden also in Triggern oder Schleifen benutzt. Sie können die Audio Datei auf irgendeine der vier Ebenen legen, wodurch es Ihnen möglich ist, dass zum Beispiel nur im Pass oder Fail Status die Audio Datei abgespielt wird.
+**Audio-Dateien** (`.wav`, `.mp3` und `.ogg`) können zu selbst gewählten Zeitpunkten abgespielt werden. Es ist wie beim Deklarieren von Objekten. Sie sind nicht als Kommandos umgesetzt und werden dadurch nicht in Triggern oder Schleifen benutzt. Du kannst die Audio-Dateien auf irgendeine der vier Ebenen legen (jedoch existiert kein Konzept der "Überschneidung", alle Sounds werden gemischt), wodurch die Audio-Dateien zum Beispiel nur im Pass- oder Fail-Status abgespielt werden.
```

Comment:
Just leaving `Pass` and `Fail` as is seems fine to me. I don't know of any German word being used for these instead.